src/components/plotly-chart/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Plot from 'react-plotly.js';
+import deepmerge from 'deepmerge';
+import { connect } from 'react-redux';
+import { dark_preview } from '../../utils/plot-templates/dark';
+import { light_preview } from '../../utils/plot-templates/light';
+import './plotly-chart.css';
+/**
+ * Display plotly chart
+ * @param {object} chartSize Chart dimensions in pixels
+ * @param {object} targetRect event.target.getBoundingClientRect()
+ * @param {boolean} visible Whether to show the tooltip
+ * @param {string} text Tooltip display label
+ */
+const PlotlyChart = ({ theme, data, layout }) => {
+  const hideToolBar = { displayModeBar: false };
+  return (
+    <div className="pipeline-plotly-chart">
+      <Plot
+        data={data}
+        layout={updateLayout(theme, layout)}
+        config={hideToolBar}
+      />
+    </div>
+  );
+};
+
+PlotlyChart.defaultProps = {
+  data: {},
+  layout: {},
+};
+
+const updateLayout = (theme, layout) => {
+  const template = theme === 'light' ? light_preview : dark_preview;
```

Comment:
Check the style guide but `light_preview` and `dark_preview` should be `LightPreview` and `DarkPreview`.

Also consider naming with a little more detail on what this is e.g. `LightPreviewTemplate`